src/main/kotlin/org/rust/ide/actions/RustfmtFileAction.kt

```diff
@@ -55,7 +50,7 @@ class RustfmtFileAction : DumbAwareAction() {
             // #1131 - Check if we get a "'rustfmt' is not installed" and let the user know to install fmt
             if ("'rustfmt' is not installed" in message) {
                 val projectDir = project.cargoProjects.findProjectForFile(file)?.manifest?.parent
-                val action = if (projectDir != null) InstallRustfmtAction(projectDir) else null
+                val action = if (projectDir != null) InstallComponentAction(projectDir, "rustfmt") else null
```

Comment:
Let's use `rustfmt-preview` here to support old toolchains. I was surprised but `rustup add component rustfmt-preview` works correctly with the latest `rustup` version

Reply:
Fixed. Also have replaced `clippy` with `clippy-preview`.